src/registry.json

```diff
@@ -5257,6 +5257,37 @@
           "checksum": "Gi/COgu8L73LLMFQGrpbDZII8igqYpVTUWTyBdKWAtA="
         }
       }
+    },
+    "npm:stellar-snap": {
+      "id": "npm:stellar-snap",
+      "metadata": {
+        "name": "Stellar",
+        "author": {
+          "name": "Paul Fears",
+          "website": "https://github.com/paulfears"
+        },
+        "website": "https://stellar-wallet-demo.vercel.app",
+        "summary": "Send, receive, and interact with Stellar tokens and contracts.",
+        "description": "Stellar Wallet\n\nThis Snap allows you to manage, send, and receive stellar tokens, as well as interact with soroban smart contracts.\n\n- Full Soroban Support\n- Full Testnet Support\n- Multiple Accounts\n- Documentation https://stellar-wallet-demo.vercel.app/docs\n- Internal Stellar/Soroban Transaction parsing",
+        "audits": [
+          {
+            "auditor": "Cure53",
+            "report": "https://github.com/paulfears/StellarSnap/tree/main/audits"
```

Comment:
Maybe link to the actual report here (with a permanent link)?

Reply:
The actual link is https://cure53.de/pentest-report_kyraview-stellar-snap.pdf

I can change it, you will have to approve again?